### DISCUSSION TITLE
batch mint rune in a single transaction

### DISCUSSION THREAD

**Discussion by Mr-six** [ordinals/ord#3488] [Q&A] [answered]
Is it possible to mint multiple runes in one transaction?

**Comment by AIRBLOOD** [comment]
@raphjaph

**Comment by raphjaph** [comment, marked as answer]
No, only one mint per tx

**Reply by raphjaph** [reply]
The underlying reason is that mints are an open distribution mechanism that is costly. The cost and restriction are the point in this case.

**Reply by abo-L** [reply]
@AIRBLOOD

**Reply by abo-L** [reply]
> Why is this not possible?
